noize_image_client:
  visualize_image_service_name: visualize_image_service
  small_max_count: 150
  big_max_count: 40

noize_image_make:
  image_height: 1544
  image_width: 2064
  divide_threshold: 100